prow/plugins/label/label.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const pluginName = "label"
+
+var (
+	allowedPrefixes = []string{"area", "priority"}
+	regexBuilder    = `(?m)^/%s[\t\s]+(?:\S*(?:[^\n]+))*`
```

Comment:
I think I'd prefer to capture each label individual, but I don't think you can write a regex to match and capture a variable number of groups (variable capture groups doesn't work).  The `[^\n]` was to avoid the multiple line problem (`/priority priority/critical\n/area area/a1`) will otherwise result in a match `priority/critical\n/area`